ui/app/src/scss/typography.scss

```diff
@@ -1,6 +1,3 @@
-// Import fonts:
-@import url("https://fonts.googleapis.com/css2?family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&family=Rouge+Script&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"rel="stylesheet");
```

Comment:
Not 100% sure if this writes an import statement but moving it out to the App component to be sure.